docker/volumes/mongo-fca-low/scripts/db-states/_default/sp.js

```diff
@@ -44,7 +45,7 @@ const fsa = {
     ],
     claims: ['amr'],
     id_token_signed_response_alg: 'HS256',
-    userinfo_signed_response_alg: 'HS256',
+    userinfo_signed_response_alg: "HS256",
```

Comment:
todo: find a way to review every commited changes one by one before commiting it. This change and the one pointed by @gdarquie can easily be avoided.

Personnaly, I add changes one by one with `git add -p` before creating a commit.